C7GameData/GameMap.cs

```diff
@@ -57,6 +57,32 @@ public void tileIndexToCoords(int index, out int x, out int y)
 			}
 		}
 
+		// Compute the outer ring of the BFC of tiles. Might not be necessary for tiles which can't be settled?
+		public void computeOuterRing()
+		{
+			// List of all the BFC tiles NOT including direct Neighbors
+			// Essentially, this is every outer tile except North->North, South->South, West->West,East->East
+			(TileDirection direction1, TileDirection direction2) [] outerRingDirections = {(TileDirection.NORTHWEST, TileDirection.NORTH),(TileDirection.NORTHWEST, TileDirection.NORTHWEST),(TileDirection.NORTHWEST, TileDirection.WEST),(TileDirection.SOUTHWEST, TileDirection.WEST),(TileDirection.SOUTHWEST, TileDirection.SOUTHWEST),(TileDirection.SOUTHWEST, TileDirection.SOUTH),(TileDirection.SOUTHEAST, TileDirection.EAST),(TileDirection.SOUTHEAST, TileDirection.SOUTHEAST),(TileDirection.SOUTHEAST, TileDirection.SOUTH),(TileDirection.NORTHEAST, TileDirection.NORTH),(TileDirection.NORTHEAST, TileDirection.NORTHEAST),(TileDirection.NORTHEAST, TileDirection.EAST)};
```

Comment:
I like this solution to the outer ring problem.  Certainly more elegant than how I did it in my editor, in a language that (at the time) didn't have tuple support.

One minor suggestion for readability would be having each of the tiles on its own line here, e.g.:

```
(TileDirection direction1, TileDirection direction2) [] outerRingDirections = {
    (TileDirection.NORTHWEST, TileDirection.NORTH),
    (TileDirection.NORTHWEST, TileDirection.NORTHWEST)
    ...
}
```

That would make it easier to verify that the expected 12 are there.  Which they are... but I did validate it just to make sure there weren't any erroneous tiles in the ring.